recipes/cairo/meson/conanfile.py

```diff
@@ -1,11 +1,15 @@
-from conans import ConanFile, tools, Meson, VisualStudioBuildEnvironment
-from conans.errors import ConanInvalidConfiguration
 import contextlib
 import glob
 import os
 
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools import files, microsoft
+from conans import tools, Meson, VisualStudioBuildEnvironment
+
 required_conan_version = ">=1.38.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
new imports, new version.